### PR TITLE
make `lever.lua` module capable

### PR DESCRIPTION
**Pull request by chdoc**
note: the removed lever descriptions are not part of the documented behavior.

### CODE REVIEW

File: lever.lua [hunk 1]
```diff
@@ -18,8 +18,6 @@ function leverPullJob(lever, priority)
 
     dfhack.job.linkIntoWorld(job, true)
     dfhack.job.checkBuildingsNow()
-
-    print(leverDescribe(lever))
```
[the remark this answers is not in the document]
Comment (chdoc) [reply lever.lua:22]: Personally, I don't think it is useful to describe the lever after pulling it. There is already separate functionality for describing levers.

[the remark this answers is not in the document]
Comment (chdoc) [reply lever.lua:22]: If you insist on preserving the printing, I would simply move the print statement after the point(s) of use within this script.

[the remark this answers is not in the document]
Comment (myk002) [reply lever.lua:22]: the output is useful as feedback to the player when calling from the CLI. When called as an API, it should be controlled by a `quiet` flag -- at least that's the pattern that I've been following for most other publicly-accessible calls like this (e.g. https://github.com/DFHack/scripts/blob/master/assign-minecarts.lua#L72)

[the remark this answers is not in the document]
Comment (chdoc) [reply lever.lua:22]: If the output was a diagnostic that related to what the function was doing, I would agree. In this case though, printing the description of the lever is only indirectly related to pulling it. In any case, I restored the behavior of the script by moving the print statements after the point of invocation.

**Review by myk002 — APPROVED** [review]

**Pull request merged**